examples/functions/fare/Fare_example_with_park_and_ride.xml

```diff
@@ -0,0 +1,170 @@
+										<Text>Park and Ride</Text>
+									</Name>
+								</LegEnd>
+								<Service>
+									<PersonalMode>car</PersonalMode>
+								</Service>
+								<Duration>PT15M</Duration>
+							</ContinuousLeg>
+						</Leg>
+						<Leg>
+							<Id>2</Id>
+							<TransferLeg>
+								<TransferType>checkIn</TransferType>
+								<TransferType>parkAndRide</TransferType>
+								<LegStart>
+									<PointOfInterestRef>myparkandride</PointOfInterestRef>
+									<Name>
+										<Text>Park and Ride</Text>
+									</Name>
+								</LegStart>
+								<LegEnd>
+									<siri:StopPointRef>8504376</siri:StopPointRef>
+									<Name>
+										<Text>the public transport stop</Text>
+									</Name>
+								</LegEnd>
+								<Duration>PT8M</Duration>
+							</TransferLeg>
+						</Leg>
+						<Leg>
+							<Id>3</Id>
+							<TimedLeg>
+								<LegBoard>
+									<siri:StopPointRef>8504376</siri:StopPointRef>
+									<StopPointName>
+										<Text>St. Petersinsel Nord</Text>
+									</StopPointName>
+									<ServiceDeparture>
+										<TimetabledTime>2021-08-20T13:40:00Z</TimetabledTime>
+									</ServiceDeparture>
+									<Order>1</Order>
+								</LegBoard>
+								<LegAlight>
+									<siri:StopPointRef>8504375</siri:StopPointRef>
+									<StopPointName>
+										<Text>Ligerz (Schiff)</Text>
+									</StopPointName>
+									<ServiceArrival>
+										<TimetabledTime>2021-08-20T13:50:00Z</TimetabledTime>
+									</ServiceArrival>
+									<Order>2</Order>
+								</LegAlight>
+								<Service>
+									<OperatingDayRef>2021-08-20</OperatingDayRef>
+									<JourneyRef>9400B:Y:H:j21:14</JourneyRef>
+									<siri:LineRef>9400B:Y</siri:LineRef>
+									<siri:DirectionRef>H</siri:DirectionRef>
+									<Mode>
+										<PtMode>water</PtMode>
+										<siri:WaterSubmode>unknown</siri:WaterSubmode>
+										<Name>
+											<Text xml:lang="de">Schiff</Text>
+										</Name>
+										<ShortName>
+											<Text xml:lang="de">BAT</Text>
+										</ShortName>
+									</Mode>
+									<ProductCategory>
+										<!-- V1.1, PR #95 - Illustration of ProductCategory -->
+										<Name>
+											<Text>Dampfschiff</Text>
+										</Name>
+									</ProductCategory>
+									<PublishedServiceName>
+										<Text>BAT</Text>
+									</PublishedServiceName>
+									<OperatorRefs>
+										<OperatorRef>182</OperatorRef>
+									</OperatorRefs>
+									<Attribute>
+										<UserText>
+											<Text xml:lang="de">Nur 2. Klasse</Text>
+										</UserText>
+										<Code>A___2</Code>
+									</Attribute>
+									<Attribute>
+										<UserText>
+											<Text xml:lang="de">Restaurant</Text>
+										</UserText>
+										<Code>A__WR</Code>
+										<Facility>
+											<siri:RefreshmentFacility>restaurantService</siri:RefreshmentFacility>
+										</Facility>
+									</Attribute>
+									<OriginText>
+										<Text>St. Petersinsel Nord</Text>
```

Comment:
```suggestion
										<Text>Biel/Bienne (Schiff/bateau)</Text>
```

Reply:
I'm not sure whether this element really should be the first stop of the *complete* service...